workers/unity/Packages/com.improbable.gdk.core/Components/WorkerEntityComponents.cs

```diff
@@ -3,25 +3,31 @@
 namespace Improbable.Gdk.Core
 {
     /// <summary>
-    ///     Component denoting a worker entity
+    ///     ECS Component denoting a worker entity
     /// </summary>
     public struct WorkerEntityTag : IComponentData
     {
     }
 
     /// <summary>
-    ///     Component added to the worker entity immediately after establishing a connection to a SpatialOS deployment
-    ///     Removed at the end of the tick it was added
+    ///     ECS Component added to the worker entity immediately after establishing a connection to a SpatialOS deployment.
     /// </summary>
+    /// <remarks>
+    ///    This is a reactive component and will be removed in the
+    ///    <see cref="Improbable.Gdk.Core.CleanReactiveComponentsSystem"/> at the end of the frame.
+    /// </remarks>
     [RemoveAtEndOfTick]
     public struct OnConnected : IComponentData
     {
     }
 
     /// <summary>
-    ///     Component added to the worker entity immediately after disconnecting from SpatialOS
-    ///     Removed at the end of the tick it was added
+    ///     ECS Component added to the worker entity immediately after disconnecting from SpatialOS
     /// </summary>
+    /// <remarks>
+    ///    This is a reactive component and will be removed in the
+    ///    <see cref="Improbable.Gdk.Core.CleanReactiveComponentsSystem"/> at the end of the frame.
```

Comment:
I could be very pedantic now and say that you should use active voice everywhere :P

Reply:
I won't fix it unless you write a comment on every instance :P